packages/components/VotingInstruction.tsx

```diff
@@ -7,11 +7,10 @@ const VotingInstructions = () => {
         fontSize="16px"
         fontFamily="inter"
         width="377px"
```

Comment:
olha só que interessante , vi esses dias [isso](https://stackoverflow.com/questions/2757483/limiting-the-number-of-characters-per-line-with-css). acho q aq vc deixou esse `width` para garantir que o texto ficara em somente duas linhas certo? mas a questao é tem uma medida propria para isso . nesse caso aq seira `w=43ch"`(43 nesse caso aq pq contei mas testa ai tmb) . [source](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)

Reply:
Isso ai é doideira demais, alterado.

Reply:
Bate papo show hein! Pra isso que serve review!!